src/app_charts/prometheus/prometheus-robot.values.yaml

```diff
@@ -60,7 +60,7 @@ prometheus:
     logLevel: warn
     # Historical data is limited to the lower of `retention` and
     # `retentionSize`.
-    retention: "1d"
+    retention: "3h"
     retentionSize: 512MB
```

Comment:
I'm thinking of sending a followup to reduce retentionSize slightly to add some margin from sizeLimit, because IIUC if set correctly, retentionSize should protect us from disk full if the `WAL + head chunk < 512Mi`, yet in the reported case it did not. Setting retention will fix it as long as `3h history + WAL + head chunk < 512Mi`, which covers the current situation but seems slightly less safe than the first condition.

Let me know if you see a reason not to do that!

Reply:
Created https://github.com/googlecloudrobotics/core/pull/356.